client/src/components/MenuHeader.js

```diff
@@ -1,48 +1,113 @@
 import React, { Component } from 'react';
-import { Menu } from 'semantic-ui-react';
+import { Menu, Grid, Sidebar, Icon, Segment, Button } from 'semantic-ui-react';
 import { withRouter } from 'react-router-dom';
 import '../styles/HeaderNavbar.css';
+import { BrowserRouter as Router, Route, Switch, Redirect } from 'react-router-dom';
+import ScoreForm from '../components/scoreForm/ScoreForm';
+import Login from '../components/Login';
+import FourOFour from '../components/404';
+import ScoreSheet from '../components/ScoreSheet';
+import Rules from '../components/Rules';
+import ProfileForm from '../components/ProfileForm';
+import HeadToHead from '../components/HeadToHead';
 
 class MenuHeader extends Component {
-  state = { activeItem: 'score form' }
+  
+  state = { 
+  	activeItem: 'score form',
+  	visible: false,
+  	dimmed: false 
+  }
+
+  toggleMobileMenu = () => this.setState({ visible:!this.state.visible, dimmed: !this.state.dimmed})
 
   handleItemClick = (e, { name, to }) => {
 		this.props.history.push(to);
 		this.setState({ activeItem: name });
 	}
```

Comment:
If you insert this.setState({ visible:!this.state.visible, dimmed: !this.state.dimmed}) between lines 26 and 27 it will do the toggle. It even keeps the animation. It might be simpler to do this than to figure out a means to make toggleMobileMenu be available in handleItemClick

Reply:
Done.